modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -72,6 +73,21 @@ class BaseConvolutionLayerImpl : public ConvolutionLayer
         int ngroups = params.get<int>("group", 1);
         CV_Assert(numOutput % ngroups == 0);
 
+        if (params.has("adj_pad")) {
+            adjust_pads.clear();
+            const DictValue& adj_pad = params.get("adj_pad");
+            for (int i = 0; i < adj_pad.size(); i++) {
+                CV_Assert(adj_pad.get<int>(i) < strides[i]);
+                adjust_pads.push_back(adj_pad.get<int>(i));
+            }
+        } else {
+            if (kernel_size.size() == 2) {
+                adjust_pads.push_back(params.get<int>("adj_h", 0));
+                adjust_pads.push_back(params.get<int>("adj_w", 0));
+            } else if (kernel_size.size() == 3) {
+                adjust_pads = std::vector<size_t>(3, 0);
+            }
+        }
```

Comment:
Is it possible to use `getParameter` here as we do for kernels and strides? Perhaps, this way we need to rename `adj_pad` to just `adj`.